### DISCUSSION TITLE
Enhancement: Custom API/API Key/Model through UI

### DISCUSSION THREAD

**Discussion by Auditormadness9** [danny-avila/LibreChat#2953] [Feature Requests & Suggestions]
### What features would you like to see added?

Custom API/API Key/Model option in the [online LibreChat space](https://librechat-librechat.hf.space/)

### More details

Would be cool to also have a new service/endpoint option added in that list among the endpoints called Other where we can input our own API url, API key, as well as the model name on https://librechat-librechat.hf.space/

### Which components are impacted by your request?

UI, Other

### Pictures

Visual Concepts:
![image](https://github.com/danny-avila/LibreChat/assets/71278545/5e1105e2-3a19-4424-b8b7-bdc7d3eff5e2)
![image](https://github.com/danny-avila/LibreChat/assets/71278545/c8706023-29c6-4106-8c9b-85f5e361dfb2)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by berry-13** [comment]
this ain't the best place to ask about the LibreChat DEMO. You should open an issue in the HF demo. And as for adding new endpoints to the UI, that's gonna be quite difficult. It'll only be added in the admin panel
